### ISSUE TITLE
mcu 'idm': Command format mismatch: query_lis2dw oid=%c clock=%u rest_ticks=%u vs query_lis2dw oid=%c rest_ticks=%u

### ISSUE THREAD

**Issue by aam335** [ModularPrintingSystem/IDM#16]
Got an error 
`mcu 'idm': Command format mismatch: query_lis2dw oid=%c clock=%u rest_ticks=%u vs query_lis2dw oid=%c rest_ticks=%u
`
Details:
latest repo (commit 3638fd0aed3a5faa7893b2ad28d73b1c0e8f6c8c)
updated Canboot_1M.bin and IDM_CAN_8kib_offset_1M.bin
python modules has correct links from extras to IDM

**Comment by ckvsoft** [comment]
If you see this error, it suggests that you need to update the IDM firmware to the latest version available
Are you really up to date?

**Comment by aam335** [comment]
yes, idm flashed succesfully via flashtool.py, got an "Flash Success"

`$ sha256sum Firmware/IDM_CAN_8kib_offset_1M.bin *.py
35a14fbc06604388777e723daa31f9ec1e16b1022ca5ac55fdd3a651b97d9f58  Firmware/IDM_CAN_8kib_offset_1M.bin
7adb552f6a690736ffbde55841229cb24d1396c8336dab58cf0e78cf9e3af04a  arg_fit.py
432b7957b9257b2af8690a01d24471f82bdf4489af705578ba2078be743091b2  idm_accel.py
b94641c142862b37949c50c0c295d0c455db2deff6e9ced2421e048e532608c5  idm.py`

**Comment by ckvsoft** [comment]
@aam335 
I don't want to compete against the code from here, but I have a few small changes in my fork. Maybe you'll have better luck with that. However, I use the USB version. Question in passing. How did you connect the CAN version? Also had a CAN for testing

Klipper is the current default Python2 version 0.12.xx for me

Symbol „Von der Community überprüft“

**Comment by ckvsoft** [comment]
Yes
![grafik](https://github.com/ModularPrintingSystem/IDM/assets/16374447/5d5c05eb-0c2d-4de2-b455-a369587b2b37)
![grafik](https://github.com/ModularPrintingSystem/IDM/assets/16374447/afc0b00f-cfb6-4d8e-ae81-bb1deafaa0df)

3x Multiplus II 5000
2x jkbms
